src/components/chat/LanguageModalSelector.tsx

```
@@ -7,8 +7,10 @@ import {
   SelectTrigger,
   SelectLabel,
   SelectGroup,
+  selectItemClassName,
```

ESLint

[error] 'selectItemClassName' is defined but never used.

(@typescript-eslint/no-unused-vars)